assets/src/components/v2/gl_eink_double/line_map.tsx

```diff
@@ -15,6 +15,7 @@ const TEXT_LEFT_MARGIN = 18;
 const TEXT_TOP_MARGIN = 10;
 const VEHICLE_ICON_SIZE = 44;
 const SCHEDULED_DEPARTURE_SIZE = 192;
+const MAXIMUM_STOP_LABEL_LENGTH = 18;
```

Comment:
This const is only used to determine whether or not the line should wrap, but it's still possible to have a stop label exceed this max on a single line.  (For example, `abcdefg hi jklmnopqr stu vwxyz` will wrap like `abcdefg hi jklmnopqr\n stu vwxyz`, giving the first line 20chars.)  Maybe add clarifying comment:  `Max length before text wrapping`)